### PR TITLE
Add Comment to readme.md

### PR DESCRIPTION
**Pull request by TrustHenry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
 # boa-sdk-ts
 SDK to interface with the BOSAGORA blockchain
+Boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
+It also has a hashing function and a pre-image validation.
```
Comment (AndrejMitrovic) [inline README.md:4]:
```suggestion
It also has utility functions such as hashing and pre-image validation.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+
+## Install
+```npm install --save boa-sdk-ts```
+
+## Import the your TypeScript
```
Comment (AndrejMitrovic) [inline README.md:9]:
```suggestion
## Import the library
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
 # boa-sdk-ts
 SDK to interface with the BOSAGORA blockchain
+Boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
```
Comment (MichaelKim20) [inline README.md:3]:
```suggestion
boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
```
The name of the package is in lower case, so I'd like to write it in lower case

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+    // On Success
+})
+.catch(err =>
+{
+    // On Error
+});
+```
+
+## Testing
+```
```
Comment (MichaelKim20) [inline README.md:32]:
```suggestion
```bash
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,37 @@
 # boa-sdk-ts
 SDK to interface with the BOSAGORA blockchain
+Boa-sdk-ts is a TypeScript/JavaScript library for communicating with the Stoa API server.
+It also has a hashing function and a pre-image validation.
+
+## Install
+```npm install --save boa-sdk-ts```
```
Comment (MichaelKim20) [inline README.md:7]:
```suggestion
```bash
npm install --save boa-sdk-ts
```

```

File: README.md (hunk 1 above)
Comment (TrustHenry) [reply README.md]: Thank you. Applied the suggestion.

File: README.md (hunk 2 above)
Comment (TrustHenry) [reply README.md]: Thank you. Applied the suggestion.

File: README.md (hunk 3 above)
Comment (TrustHenry) [reply README.md]: Thank you. Applied the suggestion.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,39 @@
...
+
+## Import the your library
+```import * as BoaSdk from "boa-sdk-ts";```
+
+## Usage
+```TypeScript
+// Set URL
+let uri = URI("http://localhost").port(port);
+// Create BOA Client
+let boa_client = new BoaSdk.BOAClient(uri.toString());
```
Comment (MichaelKim20) [inline README.md:19]:
```suggestion
// Create BOA Client
let boa_client = new BoaSdk.BOAClient("http://localhost:3836");
```

Comment (TrustHenry) [reply README.md]: Applied the suggestion.

**Pull request merged**